mkosi/qemu.py

```diff
@@ -608,7 +608,6 @@ def run_qemu(args: MkosiArgs, config: MkosiConfig, qemu_device_fds: Mapping[Qemu
             stdin=sys.stdin,
             stdout=sys.stdout,
             pass_fds=qemu_device_fds.values(),
-            env=os.environ,
```

Comment:
@DaanDeMeyer I'm not sure if this had other useful side effect (besides inheriting the parent `$PATH`). Is this safe to drop?

Reply:
The idea was that while environment variables shouldn't affect the image build by default, they should be in effect when we boot the image. So I would keep this intact.